src/modules/m_flashpolicyd.cpp

```diff
@@ -113,6 +112,7 @@ class ModuleFlashPD : public Module
 				ServerInstance->Logs->Log(MODNAME, LOG_DEFAULT, error_message);
 				ServerInstance->SNO->WriteGlobalSno('a', error_message);
 				policy_reply.clear();
+				throw ModuleException(error_message + " at " + tag->getTagLocation());
```

Comment:
You can get rid of the logging, the snomask write, and the clearing of policy_reply here.